src/picturepark-sdk-v1-angular/angular.json

```diff
@@ -324,8 +324,8 @@
                 },
                 {
                   "type": "anyComponentStyle",
-                  "maximumWarning": "6kb",
-                  "maximumError": "10kb"
+                  "maximumWarning": "18kb",
+                  "maximumError": "30kb"
```

Comment:
Did you get the new values from somewhere or did you decide on them because you thought this  size would be enough?